src/app/category-page/category-page.component.ts

```diff
@@ -119,4 +120,18 @@ export class CategoryPage implements OnInit, OnDestroy {
     this.subscriptions.forEach((sub) => { sub.unsubscribe(); });
   }
 
+  public skipToFilterSec(): void{
```

Comment:
could we avoid abbreviating here? I assume Sec = Section, but it's not clear. Probably worth the 4 letters

Reply:
@willschoener  updated in the next commit `dd77c08`.